CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Dropping a requirement of a major version of a dependency is a new contract.
 - Only one thread can update `Throttled Cache` values
 - Remove all assumptions from `DualConnection#close` and always close all sub-connections
 - Fix connection leak
+- Close replica connection on `ReplicaConsistency#isConsistent` exception
```

Comment:
This time, a couple of changes will be hidden behind `compatibleWithPreviousVersion`

Reply:
I wonder whether this change shouldnt be released as a separate. What side effects do you expect?

Reply:
I expect no side effects for us, but it can be helpful for other library consumers.